src/components/schemas/MarketStressValue.yaml

```
@@ -0,0 +1,8 @@
+type: string
+enum:
+  - VERY_TIGHT
+  - TIGHT
+  - BALANCED
+  - LOOSE
+  - VERY_LOOSE
+description: Valeurs de la tension du marché (VERY_TIGHT x>=50, TIGHT 10<=x<50, BALANCED -10<x<10, LOOSE -50<x<=-10, VERY_LOOSE x<=-50)
```

:sparkles: suggestion: Remplacer `x` par `score` pour comprendre le lien et utiliser les caractères ≥ et ≤ :
```suggestion
description: Valeurs de la tension du marché (VERY_TIGHT: score ≥ 50, TIGHT: 10 ≤ score < 50, BALANCED: -10 < score < 10, LOOSE: -50 < score ≤ -10, VERY_LOOSE: score ≤ -50)
```